src/styles/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin grid($number, $min-width) {
+  @media (min-width: $min-width) {
+    .catalog {
+      grid-template-columns: repeat($number, 200px);
+    }
+  }
+}
```

Comment:
this is an unreusable mixin because you can apply it only for catalog, so I don't see any sense in creating such mixin